File: programmingbydoing/python/45.py
# Dice

import random

def rollDice():
    print('HERE COMES THE DICE!\n')

    dice1 = random.randrange(1, 7)
    dice2 = random.randrange(1, 7)

    print('Roll #1: ', dice1)
    print('Roll #2: ', dice2)

    diceTotal = (dice1 + dice2)

    print('The total is ', diceTotal, '!')

if __name__ == '__main__':
    rollDice()

File: programmingbydoing/c/45.c
// Dice


#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int sum = 0, counter = 1;
    
    srand(time(NULL));
    int roll_1 = rand() % (6 - 1 + 1) + 1;
    int roll_2 = rand() % (6 - 1 + 1) + 1;
    
    int array[3] = {roll_1, roll_2};
    
    printf("HERE COMES THE DICE!\n");
    
    for (int i = 0; i < 2; i++) {
        printf("Roll #%d: %d\n", counter, array[i]);
        sum += array[i];
        counter++;
    }
    
    printf("The total is %d!\n", sum);
    
    return 0;
}